capitalization_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'decorator'
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    super.capitalize
```

Comment:
- [ ] Please make sure that you are implementing the requirements properly, as per instructions you should `capitalize` the `@namable` value instead of applying the method directly on the `super` method, which will not work and will result in error.

### Requirement
![image](https://github.com/Kwamay/School_Library/assets/66476329/0fdb6542-967c-43dc-8259-9b914b04d21a)